src/components/main/MovieMenu/index.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import styles from '../../../css-modules/main.module.css';
+import stylesModal from '../../../css-modules/modal.module.css';
+
+interface Props {
+    setIsOpen: React.Dispatch<React.SetStateAction<boolean>>,
+    setIsOpenDeleteModal: React.Dispatch<React.SetStateAction<boolean>>,
+    setIsOpenEditModal: React.Dispatch<React.SetStateAction<boolean>>
+}
```

Comment:
usually it's better to use this:
```suggestion
interface Props {
    setIsOpen: (newState:boolean) => void,
    setIsOpenDeleteModal: (newState:boolean) => void,
    setIsOpenEditModal: (newState:boolean) => void
}
```
cause child component don't really need to know what the passed function do in parent component

Reply:
fixed